indico/modules/events/tracks/templates/display.html

```diff
@@ -24,9 +24,15 @@
     <ul class="programme">
         {% for track in event.tracks %}
```

Comment:
Im getting this error : 
ArgumentError: Can't find property named 'reviewers' on the mapped entity Mapper|Track|tracks in this Query.